css/style.css

```diff
@@ -203,8 +206,8 @@ aside h2 {
 /* ----------- UI - Top Bar ----------- */
 
 main {
-  display: grid;
-  grid-template-rows: 10% 60% 30%;
```

Comment:
This is actually one of the few areas where I think flexbox is more appropriate for sure.

The reason why I changed this to `display: flex` is because otherwise there was uneven whitespace underneath the toggles and filters relative to the rest of the main content area. Giving a height for the footer of 30% instead of using flexbox creates this unevenness. Even using fractional units (ex: 1fr with grid) still says: Make this element X relative to the size of other elements/the height of the page. What we really want is to say: place it as far down the page as possible given its size. `display: flex`, `flex-direction: column` and `justify-content: space-between` allows us to do that. Doing this allows us not only to have even whitespace around the main content area, but also that extra space the footer was taking up in grid is now used to make the visualization area larger.

Before:
![screen shot 2017-08-16 at 8 28 29 am](https://user-images.githubusercontent.com/17437436/29371106-4fc3e5f8-825c-11e7-8403-b3b7266fca67.png)

After (with `.unimplemented` class removed on footer)
![screen shot 2017-08-16 at 8 31 07 am](https://user-images.githubusercontent.com/17437436/29371227-ac6ab732-825c-11e7-95c4-3fa1f49dfeb1.png)